doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -2961,6 +2961,8 @@ Example: ``curl -H "X-Dataverse-key: $API_TOKEN" -X POST http://demo.dataverse.o
 
 This action moves account data from jsmith2 into the account jsmith and deletes the account of jsmith2.
 
+Note: User accounts can only be merged if they are either both enabled or both disabled. See :ref:`disable a user<disable-a-user>`.
```

Comment:
Merging disabled accounts into each other seems rare but I suppose we could support it. If it's problematic I'd suggest deprioritizing it.